### PR TITLE
Synced local files to container files

### LINKED ISSUE #167: Change in docker compose.yaml file
### Description

As there is no bind or docker compose watch present the local developed code does not sync automatically with the container files. 

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by PadamSinha2511**
Fixes Issue: -  Change in docker compose.yaml 

Closes  #167

## Changes proposed

Added a volume over the 
Local files to container files i.e. .:/usr/src/app
And another for node modules so that they don't overlap i.e. /usr/src/app/node_modules

### CODE REVIEW

**Review by Bashamega — APPROVED** [review]

File: compose.yaml [hunk 1]
```diff
@@ -3,6 +3,9 @@ services:
     build:
       context: .
       dockerfile: Dockerfile.dev
+    volumes:
+      - .:/usr/src/app
+      - /usr/src/app/node_modules
```
Comment (eddiejaoude) [inline compose.yaml:8]: Is `/usr/src/app/node_modules` this needed? Because it will be mounted as part of the project on `.`?

Comment (Pradumnasaraf) [reply compose.yaml:8]: Hey @eddiejaoude,

The above config is correct. Both are different. This one, `.:/usr/src/app`, mounts the entire current directory. Second, we’re not mounting again; instead, we’re preventing `node_modules` from being overridden in the container and the container uses its dependencies using `/usr/src/app/node_modules`.

Comment (eddiejaoude) [reply compose.yaml:8]: oh I see, thank you for explaining 👍

**Review by Pradumnasaraf — APPROVED** [review]
Looks Good to merge!

**Comment by eddiejaoude** [comment]
note: sorry for the delay in merging, I was clearing up disk space on the server (we ran out so deployments were no longer working 🤦‍♂️ , not sure how we used 100GB in 1 week 😰 )

**Pull request merged**